lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -246,16 +254,17 @@ def self.upload(app_id, binary_path)
               UI.message("Uploading the APK/IPA")
               upload_binary(app_id, binary_path)
             end
-            status = upload_status(upload_token, app_id)
+            status, release_id = upload_status(upload_token, app_id)
```

Comment:
On cases of a failure the release_id is nil which is ok. In our other PR https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/pull/60 we have handled the case where the method would conclude not on a success if it reached the maximum tries.

Reply:
Can you document this in a comment above the upload_status method? I think it would be good to document any method where the behavior, arguments, and return value aren't immediately clear from the signature.